src/api/user.controller.js

```diff
@@ -0,0 +1,78 @@
+const { userService } = require('../services/user.service.js');
+
+const getUsers = async (req, res) => {
+  const users = await userService.getUsers();
+
+  res.json(users);
+};
+
+const createUser = async (req, res) => {
+  const name = req.body.name;
```

Comment:
Consider adding validation for other potential fields in the `req.body` if they are expected in the `createUser` function. This will ensure that all necessary data is validated before proceeding.